plugins/woocommerce/src/Admin/RemoteInboxNotifications/Transformers/PrepareUrl.php

```diff
@@ -14,15 +14,32 @@ class PrepareUrl implements TransformerInterface {
 	/**
 	 * Prepares the site URL by removing the protocol and trailing slash.
 	 *
-	 * @param mixed         $value a value to transform.
+	 * @param string        $value a value to transform.
 	 * @param stdClass|null $arguments arguments.
 	 * @param string|null   $default default value.
 	 *
 	 * @return mixed|null
 	 */
 	public function transform( $value, stdClass $arguments = null, $default = null ) {
+		if ( ! is_string( $value ) ) {
+			return $default;
+		}
+
 		$url_parts = wp_parse_url( rtrim( $value, '/' ) );
```

Comment:
https://developer.wordpress.org/reference/functions/wp_parse_url/

> [Return](https://developer.wordpress.org/reference/functions/wp_parse_url/#return)
mixed **False** on parse failure; **Array of URL** components on success; When a specific component has been requested: null if the component doesn’t exist in the given URL; a string or – in the case of PHP_URL_PORT – integer when it does. See parse_url()’s return values.